CHANGELOG.md

```diff
@@ -0,0 +1,91 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.3.0] - 2020-03-30
+
+### Added
+
+- Added `CHANGELOG.md`
```

Comment:
```suggestion
```

"The purpose of a changelog entry is to document the noteworthy difference, often across multiple commits, to communicate them clearly to end users. "

I don't consider a changelog "noteworthy", because it doesn't impact our users :P